index_matroska.md

```diff
@@ -64,7 +64,7 @@ Matroska is an open standards project. This means for personal use it is absolut
 
 # Status of this document
 
-This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). But since it's quite complete it is used as a reference for the development of libmatroska.
+This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). But since it's quite complete it is used as a reference for the development of libmatroska. Legacy versions of the specification can be found [here](https://www.matroska.org/files/matroska_file_format_alexander_noé.pdf) (PDF doc by Alexander Noé -- outdated).
```

Comment:
Nit: why is “legacy versions” plural? To me the PDF is one working version.

Reply:
Yeah. I just reversed the commit that got rid of the line, but I agree that singular's better.

Reply:
Ugh, our generate-specs-process doesn't allow non-ASCII characters in hrefs.

Reply:
I guess, you can encode it with %nm (I have to check the numbers), like in the old times.

Reply:
Nah, I just decided to use `…noe` instead.